spot-client/src/zoom/iframe-api/IframeApi.js

```diff
@@ -76,6 +76,16 @@ export class IFrameApi {
 
             break;
         }
+
+        case commands.SUBMIT_PASSWORD: {
+            sdk.submitPassword(data.password)
+                .then(
+                    () => this._sendMessageToParent(events.MEETING_JOIN_SUCCEEDED),
+                    error => this._sendMessageToParent(events.MEETING_JOIN_FAILED, { error })
```

Comment:
what is the `error` in this case? could it be anything?

Reply:
It is the reject from sdk.joinMeeting. It's usually the errorCode on failed meeting join, but it could be an error from something else in the join chain.

Reply:
I see that it's ok because there's a generic error handler